src/xdg.c

```diff
@@ -393,15 +350,15 @@ xdg_toplevel_view_move_to_front(struct view *view)
 {
 	struct view *root = xdg_toplevel_view_get_root(view);
 	view_impl_move_to_front(root);
-	move_sub_views(root, LAB_TO_FRONT);
+	view_impl_move_sub_views(root, LAB_TO_FRONT);
 }
 
 static void
 xdg_toplevel_view_move_to_back(struct view *view)
 {
 	struct view *root = xdg_toplevel_view_get_root(view);
 	view_impl_move_to_back(root);
-	move_sub_views(root, LAB_TO_BACK);
+	view_impl_move_sub_views(root, LAB_TO_BACK);
```

Comment:
Should we switch these two around for the `_to_back()` case, e.g. first `move_sub_views()` and then `move_to_back(root)`?
Situation: dialogs on top of toplevel.

Currently:
- toplevel to the back
- dialog to the back
- result: dialog behind toplevel

It will fix itself on unminimize but might mess up A-Tab ordering and similar.